showpass/Showpass-E2E-tests
# Verify the right hand top drop down list items

**Description:**
Once a user logs in, they will have a list of menu items available under the User Name.
Following are the items to be verified:

- Stop Impersonate (might not show up)
- Upcoming
- Orders
- Memberships
- Credits
- Products
- Saved Events
- Profile
- Support
- Log Out

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-11

**How to test:**
Run this spec and ensure it passes.
it("verifying drop-down items under username after logging in", () => {
    cy.readFile("cypress/fixtures/testdata.json").then((userObject) => {
      cy.logIntoPortal(userObject.userDetails);
      cy.clickUsernameAfterLoggingIn();
      cy.verifyDropDownItemExists(userObject.topRightHandDropDownList);
    });

**Test execution results:**
<img width="1716" alt="rightHandDropDown-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/cbf78fa8-1d82-49f9-85f4-d1cb43718010">
